### PR TITLE
Modifying test_target_map_pointer.default.c to use headerfile, as per…

### PR DESCRIPTION
**Pull request by tmh97**
This test checks that the default behavior of map clause is tofrom when no map-type is specified for pointer p.

Fixed macro usage - > issue #10 

Passes on Fatnode for all relevant GCC and Clang versions.

### CODE REVIEW

File: tests/4.5/target/test_target_map_pointer_default.c [hunk 1]
```diff
@@ -9,45 +9,51 @@
...
+  int i;
  
   for (i = 0; i < N; i++) 
     compute_array[i] = 0;
+  
   p = &compute_array[0];
 
-// p is mapped as a pointer to compute_array
+// p is default mapped tofrom as a pointer to compute_array
 #pragma omp target data map(tofrom: compute_array)
```
Comment (tmh97) [inline tests/4.5/target/test_target_map_pointer_default.c]: Remove line 32 for default mapping, must keep this when testing implicit mapping

**Comment by tmh97** [comment]
Added additional tests for default pointer behavior, passes all GCC and Clang versions

**Comment by tmh97** [comment]
Split the file into two separate tests, re-ran both and they pass on relevant Clang and GCC versions.

File: tests/4.5/target/test_target_map_pointer_default.c [hunk 2]
```diff
@@ -1,53 +1,58 @@
-//===--- test_target_map_pointer.c - test default behavior pointer mapping --===//
+//===--- test_target_map_pointer_default.c -------------------------------------===//
```
Comment (josemonsalve2) [inline tests/4.5/target/test_target_map_pointer_default.c]: Rename this file to `test_target_map_pointer_no_map_type_modifier`

**Comment by josemonsalve2** [comment]
After changing the file name it should be good to go

**Comment by josemonsalve2** [comment]
If it runs, and there are no bugs, then go ahead and merge it.

**Pull request merged**